samples/metaapp/src/main/java/com/sourcepointmeta/metaapp/tv/cards/PropertyCardView.kt

```diff
@@ -0,0 +1,35 @@
+package com.sourcepointmeta.metaapp.tv.cards
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.widget.TextView
+import androidx.leanback.widget.BaseCardView
+import com.google.android.material.chip.Chip
+import com.sourcepointmeta.metaapp.R
+
+open class PropertyCardView(context: Context?)  : BaseCardView(context) {
+    var propertyNameView: TextView? = null
```

Comment:
we need to rework this card in the next MR, not now

Reply:
Okay, will do. 
It would be great to have a task with a short text description (in the form of a user story) of what to rework.
Should I squash and merge into `develop` now?